task-assembly/pom.xml

```diff
@@ -31,8 +32,10 @@
                             <goal>single</goal>
                         </goals>
                         <configuration>
+                            <skipAssembly>${assembly.skip}</skipAssembly>
                             <finalName>${assembly.name}</finalName>
                             <appendAssemblyId>false</appendAssemblyId>
+                            <attach>false</attach>

```

Comment:
Avoid distributing binary packages.

Without it:

![111](https://github.com/cubefs/compass/assets/17894939/51a99449-015d-424e-8efe-29bc09823fcd)

Reply:
Yes, we do not need task-assembly package.